source/main/locales/en-US.json

```diff
@@ -2,35 +2,35 @@
   "menu.daedalus": "Daedalus",
   "menu.daedalus.about": "About Daedalus",
   "menu.daedalus.adaRedemption": "Ada Redemption",
-  "menu.daedalus.daedalusDiagnostics": "Daedalus Diagnostics",
-  "menu.daedalus.blockConsolidationStatus": "Block Consolidation Status",
-  "menu.daedalus.quit": "Quit",
   "menu.daedalus.close": "Close",
+  "menu.daedalus.quit": "Quit",
   "menu.edit": "Edit",
-  "menu.edit.undo": "Undo",
-  "menu.edit.redo": "Redo",
-  "menu.edit.cut": "Cut",
   "menu.edit.copy": "Copy",
+  "menu.edit.cut": "Cut",
   "menu.edit.paste": "Paste",
+  "menu.edit.redo": "Redo",
   "menu.edit.selectAll": "Select All",
-  "menu.helpSupport": "Help and Support",
-  "menu.helpSupport.gpuSafeMode": "GPU Safe Mode",
-  "menu.helpSupport.gpuSafeModeDialogTitle": "Turn off 'GPU safe mode'?",
-  "menu.helpSupport.nonGpuSafeModeDialogTitle": "Turn on 'GPU safe mode'?",
-  "menu.helpSupport.gpuSafeModeDialogMessage": "Turn off 'GPU safe mode'? \n \nYou are about to disable 'GPU safe mode' which will improve the performance of user interface rendering. \n \nClick on 'Yes' to restart Daedalus with 'GPU safe mode' disabled. \n \n If you see a blank screen instead of Daedalus user interface after Daedalus restarts, please turn 'GPU safe mode' back on.",
-  "menu.helpSupport.nonGpuSafeModeDialogMessage": "Turn on 'GPU safe mode'? \n \nYou are about to enable 'GPU safe mode' which will reduce the performance of user interface rendering. \n \nYou should enable this only if you are seeing a blank screen instead of Daedalus user interface. If Daedalus is working properly for you, please click the 'No'. Otherwise, click 'Yes' to restart Daedalus with 'GPU safe mode' enabled.",
+  "menu.edit.undo": "Undo",
+  "menu.helpSupport": "Help",
+  "menu.helpSupport.blockConsolidationStatus": "Block Consolidation Status",
+  "menu.helpSupport.daedalusDiagnostics": "Daedalus Diagnostics",
+  "menu.helpSupport.downloadLogs": "Download Logs",
+  "menu.helpSupport.blankScreenFix": "Blank screen fix",
```

Comment:
@daniloprates this `Blank screen fix` should be `Blank Screen Fix` 🙏

Reply:
Fixed in: https://github.com/input-output-hk/daedalus/pull/1459/commits/a6514e197a57aa98b50d5c6ccd6f2ddce5bfe49b